corehq/apps/accounting/migrations/0065_phone_apk_heartbeat_privs.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 1.11.21 on 2019-11-27 11:11
+
+from django.core.management import call_command
+from django.db import migrations
+
+from corehq.privileges import PHONE_APK_HEARTBEAT
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def _grandfather_phone_apk_heartbeat_privs(apps, schema_editor):
+    call_command('cchq_prbac_bootstrap')
+    call_command(
+        'cchq_prbac_grandfather_privs',
+        PHONE_APK_HEARTBEAT,
+        skip_edition='Community',
```

Comment:
Shouldn't `Paused` be here as well?

Reply:
@sravfeyn I'll make the update, but what is the `Paused` edition exactly?

Reply:
I think this one https://confluence.dimagi.com/display/saas/CommCare+Subscriptions+-+Auto+Pause+Process https://confluence.dimagi.com/display/saas/CommCare+HQ+Accounting+-+Subscription+Creation+and+Modification